deploy/complete/terraform/schema.yaml

```diff
@@ -7,12 +7,14 @@ description: "Oracle Container Engine (OKE) stack for MuShop App"
9k9PxLkQa8PAOZdNeUvyRBakHoYi1ck6XoOb/GHdqICr9zfEU8qDlNm5iUnHPiVv4YmMAS86eFEJQpcpu3OEUV2ElYJEWMbsRo5kzcfVyYoCYp93gj5pRFjCbKIZilj92LXWMPsnpmSHbCWcZx34tr47IRVRO8TskqIf87UDsncqB87s/CRmw8qqXknSoMmQdpWVOCturWooutt5A/j7HHvlbEFSrQJ/nSNh7y5wLtvPMwre1SaC25nDHDJkbUXrY3bwVpUAZaVEWPK81m2z3s1KQZ9/t2JTr1nH5qgcRwhSdK9N3nB3QFFgFkYpqiMq+buyUqHo969xVON32f76B03CelqVl/PDDBEfxELBA67eEhcWIESa8ZenHqIEgmZdx104KrrGcNSJ4NXJQt4NcDPFqUuEJDsh/m4DNkl1lIKcaGRUwL2L1zkYkrwDK60jymQ6P0/vqyVCy7sV01IJdl9lI0RII0WSiB9wdWm9eG3FIGWg0843TIgzuLuQ0ghhE14GFIu1B5jkmEAdt1VjEKacqN5DmEWiNOlyQmHjNXkcyxkDe608r/vkHmYFu35cv+kXjIcUJkKaqxjFvANXL8X3SAQZkiHJZhhmxAPipEhLv3c3wOYRYVbv6zWJcFARvTfpfEsZRDijyOEVSaE0IhwC3ohZjHqK7BXZUCQ4oHtZaloiRAXdMlc6BDFRTYa7OcyQEmlBoyQihAkqIs9WCNR5TRBhUoGpw8CvYHYyr6Zc85ckFWaALysqmSatXi6yzpOdS2lzP7u6jyfHqRqyrPogwbgragyW9RTipAtT1FARW8GGiTepl5CyoRluw882Myt+zhmRZVKD91P6ezOhXfMiQaQZntW+E/qutQ8WmggxZ3V+RPaScdJKYtbiUtFzpr1snKOvmWDD2UUWngInhtCvSYQVTD4mmLWFt4sQ8Qn3Hn3eESluwJTKibfrBkwNpVAkO59jzJ6SKlmVfbI+DomQNZuDFMOvqbzHNNJGOS5qmcgi1E/K2cCsCXxUdsKdMggfA/5LM/yX9VlXgxpoJn8I+DXMLuoAk9h7WoRax1RNyavVdJ2+f9Eh5LYQIY0QdQJCTZMjrx12HcWt0VgHPeAD0tVn5EXM6dyhyNAFPgP8o2bwHQpqvUKDb6XUBKMaSnfnxDeOMlrh7IkM206EpqXAuEmTp/rqYCAj7QOYVUSbNTSpYwuYIhoP6/gB6f6XyU08qDZckXj/PCZBpZdD5lNyX22hjqd2kghpBuS4Bi7I+V1UQaI1QeSBBm8J+HfHMBxIfz/qGH52kesSptSvzeJyE1aLLIu/UNJgBfgW+XWXxk6EOgNb59xByWtaleAuzHQalmiLjCqk5tkW5/WqijndzyI110pC2o0iq4xuZbNJuVoneHZtiDZiVve1XtVb2CtESJv1NsYwj8n3e6kG/5DjLcTrObfxPm06/iY110v2GhE6GvCbFeU7QfLimC2E1ZPrttzS+41q2GV7kghdBWJukb8+xyhxdk1+/pPAN7h6J9JA/2+0lAQ2eJZU1HNfEaEjq/s2uVIn5IrZcvuPY/ZSLsuYWtNrk73xGMEVkWGaiplJbSeCff7haUydhVvVET25cF+TC/c4o4zqvYhFSbYZrq73vC+I0JXovxETk79Rov2SSPBp+ewb7H18Q9LPVnAN9zDpn6UKTmJi8E8DPwT+B/gUJlQ7xf7C9ZjSvc9gNsNM75cbP43ZEfQDkeB7mGrpUy2OBdTBAXk9z2gynN4PN30rZrv5D/R6WFLgwD4lgbWV7pVU/CJm9XPP2ggTcg3/BLOq18fkBn6Y5K3i+wl2oWpDE+LQXiXChFzCd8tNXMNsP78Xs96/hce3HSI8t45o2c3ewXF5Bq/UoJ8FPkL6PsD9iJ5DhNm9JhHsTuXbgN/SZ49iHod7kfYUvdouItjHJ81UdSF3q0SYwCwW3SmXsYd5UPYlT4JnYUVECBk9u3vPqIbbMU9yPygSvJHsjSP7GRtOv3SouQl2N+Fa4DUyDhcVKDnP3lgbGBcRBs54Tu0FiTCBeQjmKYm7xzBFN7wkSMeW0z/hXlANISYD+EYZPReAT1AjIXOfwC0paJ8g11oi2FTuOxQU6WEih5/zKqEQ4s/IaC0R3MyiScwq4iPUzMPzaB8RpjGFJydl/Pwb9bJ7PVroNUxKGrxW7blbJPAG4j6TCPOYCKKtXvKANxBLj6FbLabfRiKEmFDyK3UTnyG70ITHs2H3VdoamWttJMKcYgbzcoMeYGefrtpWIkw4ruRqG4lwDFNFBKkDv6pYHlOM6l8Pq3paO0kE+/ALG0V8iOzaxR7JmJFEsPbBRtuIMAf8DGZhaUNqwZOgOhHij0lqDRGulUToYzagnPdEqIQjmDjMGjWe6bSTRHgxpmLICjWeReTBT0sqrGJK9bSKCLOMKpD1MBVC/JpCedjNPgckERbbRoTjjApe9jD7FD3Kwz7VpdNWIpxk9PS3JdpdIn+nEGhCueV5lttEhAD4OTF5GfOAD68WquE6Rlv/v0ONPR47QYRQEmFa0mDBj2flsTujflyo24/BDjT+hBofYnbpeCJUm0yHpRom2kiEEJOU2sUEkb5Fu5+cspPewqsZFeJ+oo6huFMS4Wd1Az1GFT88ymECs3TfwRQFuUzNNZqdIIItTL1MxZUyTwKukWQNMSl9i3UNbk+E9uEg8CrMquM6Zjv8lSYGZjtJMIGJhHkiVLex5jEPKAez72OBBsoEbScROhJptgycNxTLYw5TK+K4/v8IDQXjwm1ms638ZZ9n5NPVy03a64G3MHpu9Geb6sPtJEJfouzPMU8iu4CPKJbBPPAixQ82MSUCGqsXuZ1EGDJ6gPYmPjexLAbA9zHR2IuYRN/GXO/nbPPNPIOphPZ9fP5BWfwQ+D/gm5itgF9Wf3p4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHh4eHi0Ff8PIS+RFSylw7QAAAAASUVORK5CYII=
```

Comment:
I'll work on `svg` for you

Reply:
Thanks.

Reply:
thanks @mvandervliet for the svg. Converted to data uri base64 and updated to the schema. The reduction was not much, but was something.
<img width="485" alt="image" src="https://user-images.githubusercontent.com/551007/82574224-67d42e80-9b4c-11ea-96b9-0cd7c0d41e9a.png">